.circleci/config.yml

```diff
@@ -125,12 +124,29 @@ jobs:
           name: Push Docker image to Quay
           command: |
             docker login -u $DOCKER_USER -p $DOCKER_PASS quay.io
-            docker tag quay.io/plotly/image-exporter:$CIRCLE_SHA1 quay.io/plotly/image-exporter:$CIRCLE_BRANCH
             docker push quay.io/plotly/image-exporter:$CIRCLE_SHA1
-            docker push quay.io/plotly/image-exporter:$CIRCLE_BRANCH
-            if [[ ! -z "${CIRCLE_TAG}" ]]; then
-              docker tag quay.io/plotly/image-exporter:$CIRCLE_SHA1 quay.io/plotly/image-exporter:$CIRCLE_TAG
-              docker push quay.io/plotly/image-exporter:$CIRCLE_TAG
+
+            # Is this a "version" commit? e.g. from `npm version`
+            # inspired by: https://discuss.circleci.com/t/git-tag-deploys-in-2-0/9493/4
+            if git log -1 --pretty=%B | grep "^[0-9]\+\.[0-9]\+\.[0-9]\+$"; then
```

Comment:
This regex wouldn't match anything starting with a v. I think it should, shouldn't it?

Reply:
... but `npm version` writes version commits w/o a leading `v`

Reply:
Right, git tags start with `v`, but version commits like this one:

https://github.com/plotly/orca/commit/d50563f4d6dbc20c4a5a37e15c9465f90e97a37a

don't

Reply:
Oh wow, I was really confused. `git log --pretty=%B` actually prints out the commit message NOT the git tag. This means a commit message that looks like a version number can screw things up but I guess that won't happen.

Non-blocking but the [release checklist](https://github.com/plotly/orca/blob/master/CONTRIBUTING.md#checklist) states that running `npm version` does a `git commit` with message `vX.Y.Z`. Actually the message for version commits doesn't have a leading `v` and has a message `X.Y.Z`. Maybe fix that typo in `CONTRIBUTING.md` for clarity.

Reply:
> Maybe fix that typo in `CONTRIBUTING.md` for clarity.

Good call. Thanks! Done in -> https://github.com/plotly/orca/commit/0afcec0e15be168f77f91b60750a03a7476ffa89